### PR TITLE
feat: add arrows prop

### PR DESCRIPTION
**Pull request by suleiy**
### 🔗 Linked issue

https://energysage.atlassian.net/browse/CED-667
### ❓ Type of change

- [ ] 📖 Documentation (updates to the documentation or readme)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [x] ✨ New feature (a non-breaking change that adds functionality)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

In ArticleRelated desktop design, it has arrows to navigate through the horizontal list. Added a prop to have the option to use arrows in desktop. 

### 🥼 Testing

Manual Testing (es-design-system/pages/molecules/es-horizontal-list.vue)
### 📝 Checklist

- [x] I have linked an issue or discussion.
- [x] I have updated the documentation accordingly.
- [x] I have documented testing approach

### CODE REVIEW

**Review by lgeggleston — APPROVED** [review]
This looks very nice! True to the design and functionality; Tested on multiple viewports and with a non-center aligned version, all look as expected and no change to previous versions.

File: es-vue-base/src/lib-components/EsHorizontalList.vue [hunk 1]
```diff
@@ -102,6 +124,17 @@ export default {
             this.activeIndex = index;
             scroller.scrollLeft = element.offsetLeft;
         },
+
+        arrowClick(direction) {
+            const scroller = this.$refs[this.id];
+            const indexChange = direction === 'left' ? -1 : 1;
+
+            if (this.activeIndex + indexChange >= 1 && this.activeIndex + indexChange <= this.children.length) {
```
[the remark this answers is not in the document]
Comment (suleiy) [reply es-vue-base/src/lib-components/EsHorizontalList.vue]: Fixed, thank you!

File: es-vue-base/src/lib-components/EsHorizontalList.vue [hunk 2]
```diff
@@ -24,7 +24,14 @@
...
             v-if="dots"
-            class="dots-holder list-unstyled w-100 mt-100 text-center">
+            class="dots-holder list-unstyled w-100 mt-100 text-center d-flex align-items-center justify-content-center">
+            <es-button
+                v-if="arrows"
+                variant="link"
+                class="text-black d-none d-lg-block p-0 mr-200"
+                @click="arrowClick('left')">
+                <icon-chevron-left />
+            </es-button>
```
Comment (ericdouglaspratt) [inline es-vue-base/src/lib-components/EsHorizontalList.vue:34]: Can we throw a `<span class="sr-only">Previous</span>` in here for accessibility purposes, so the function if this button is clear to screen readers?

File: es-vue-base/src/lib-components/EsHorizontalList.vue [hunk 3]
```diff
@@ -35,6 +42,13 @@
                     active: index === activeIndex,
                 }"
                 @click="dotClick(index)" />
+            <es-button
+                v-if="arrows"
+                variant="link"
+                class="text-black d-none d-lg-block p-0 ml-200"
+                @click="arrowClick('right')">
+                <icon-chevron-right />
+            </es-button>
```
Comment (ericdouglaspratt) [inline es-vue-base/src/lib-components/EsHorizontalList.vue:51]: Can we throw a `<span class="sr-only">Next</span>` in here for accessibility purposes, so the function if this button is clear to screen readers?

**Review by ericdouglaspratt — CHANGES_REQUESTED** [review]
This is great! It makes EsHorizontalList actually functional for mouse-based users. Two small accessibility changes and then I'd say this is good to go.

**Review by ericdouglaspratt — APPROVED** [review]
Nice work!

**Pull request merged**